### PR TITLE
New explore recipe

### PR DESCRIPTION
**Pull request by pinyaoting**
@ShawnDuan @santhoshsachindran 

New Design on Recipe page. Please review & merge this after PR #103 is merged.

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (santhoshsachindran) [inline app/src/main/res/values/styles.xml:4]: Why are you choosing NoActionBar here ? I thought we are going with ToolBar for everypage

Comment (pinyaoting) [reply app/src/main/res/values/styles.xml:4]: please download the branch and run it, you'll see why

**Review by santhoshsachindran — CHANGES_REQUESTED** [review]
@pinyaoting my personal opinion i really don't like the overall look, if feels too dark, i liked the previous view.

Things i like:
The new way of how you are showing image and text is good, we can use the same design but with old overall theme.

Things i think have degraded than previous
1. As soon as i loaded the app, since you have moved the name to the top, i wasn't sure if the name was for the first (random image) being loaded or for the one below, took me a while to understand. To a user as soon as they see it should be intuitive.

2. Image seems to be not fitting in the image view, the problem is that it says yellow cupcake but i can only see frosting.

3. Looks like you made the decision to move the title to top so the search bar can stay there, i still feel the title should be at bottom as previously and search should be in toolbar.

4. Since it's a dark theme, i cannot even see what is the other option in view pager bar, it is just dark, i need to be able to see what is there.

**Comment by pinyaoting** [comment]
@santhoshsachindran well, I really don't like the previous view because it simply reminds me of every other app on android. if dark is your main concern we can change the theme to pitch white instead.

**Comment by pinyaoting** [comment]
as for the first issue you mentioned, I disagree that user would have difficulty matching title to image. Please notice I've been using vibrant color for the title so that it matches with the image naturally.

**Comment by santhoshsachindran** [comment]
@pinyaoting dark is just one of the thing that is concerning, but even others as i have mentioned,

we are showing search bar all the time even when we don't know if user is going to search or not. The name placement is confusing, i need to understand right away which one it is for. I personally felt that and was wondering and took me a few seconds to understand image follows later. I told that based on my experience.

I get your concern about it being it like other apps, my opinion is we shouldn't do something just to be different, we need to consider the user experience as well and how it will let users keep using the app.

Also with this new design where are you planning to put the sign out option ?

**Comment by pinyaoting** [comment]
we'll have a transparent toolbar, we can move searchbox there and hide it by default. the "more" button in toolbar will be white, and will be the only thing visible other then searchbox.

**Comment by pinyaoting** [comment]
and actually if you don't like the name on top, that's fine we can try leave the number on top and have title on bottom.

**Comment by santhoshsachindran** [comment]
surely we can see how that looks and then decide. I just didn't want to waste your time if we are deciding on one path vs the other. Only when i see with your new changes i can give you my feedback, so i am just trying to see that we all agree on path and move on.

**Comment by pinyaoting** [comment]
Well, I've been working on UI development for the past 5 years, so don't you think I should already know what I'm doing without adult supervision? :)

**Comment by santhoshsachindran** [comment]
I don't believe i doubt that at all :) I just want to make sure that expertise is used the best way :) i am giving my opinion as a common user !

**Pull request closed without merging**